Bio/PDB/PDBList.py

```diff
@@ -266,21 +314,55 @@ def update_pdb(self):
             else:
                 print("Obsolete file %s is missing" % old_file)
 
-    def download_entire_pdb(self, listfile=None):
+    def download_pdb_files(self, pdb_codes, file_format='mmCif', overwrite=False, obsolete=False, pdir=None):
+        """ Retrieves a set of PDB structure files from the PDB server and stores them in a local file tree.
+
+        The PDB structure's file name is returned as a single string.
+        If obsolete ``==`` True, the files will be saved in a special file tree.
+
+        @param pdb_codes: a list of 4-symbols structure Ids from PDB
+        @type pdb_codes: list of strings
+
+        @param file_format: file format. Available options:
+            "mmCif" (default, PDBx/mmCif file),
+            "pdb" (format PDB),
+            "xml" (PMDML/XML format),
+            "mmtf" (highly compressed),
+            "bundle" (PDB formatted archive for large structure}
+
+        @param overwrite: if set to True, existing structure files will be overwritten. Default: False
+        @type overwrite: bool
+
+        @param obsolete: has a meaning only for obsolete structures. If True, download the obsolete structure
+            to 'obsolete' folder, otherwise download won't be performed.
+            This option doesn't work for mmtf format as obsoleted structures doesn't stored in mmtf.
+        (default: False)
+        @type obsolete: bool
+
+        @param pdir: put the file in this directory (default: create a PDB-style directory tree)
+        @type pdir: string
+
+        @return: filenames
+        @rtype: string
+        """
+        for pdb_code in pdb_codes:
+            self.retrieve_pdb_file(pdb_code, obsolete=obsolete, pdir=pdir, file_format=file_format, overwrite=overwrite)
+
+    def download_entire_pdb(self, file_format='mmCif', listfile=None):
```

Comment:
Original API was:

``` python
    def download_entire_pdb(self, listfile=None):
```

To preserve the order, I suggest:

```
   def download_entire_pdb(self, listfile=None, file_format='mmCif'):
```

The docstring should comment on the change in default file format, which I assume was PDB format?